docs/partials/setup.md

```diff
@@ -0,0 +1,63 @@
+The `auro-sidenav` component is configured to remain in "mobile" (read: accordion dropdown) form until passed the
+`static` attribute. See the code example below for a vanilla JavaScript solution to add and remove this attribute
+automatically when resizing the browser window. To see this code in action, resize the window for the **Responsive**
+example farther down the page.
+
+_This code example is a wrapper around the native `window.matchMedia` and may not work as expected if you directly copy
+and paste this into Server Side Rendering environments. Sidenav works just fine in SRR, but this specific function has not been
+tested to run on a server._
```

Comment:
Oops, see other comments related to this content for the updated copy. 

Auro's writing style is pragmatically direct. The subject is the code itself, not the person you are 'talking' to. Avoid words like "we" and "your" and avoid casual conversation statements.